src/main/java/models/Book.java

```diff
@@ -36,7 +36,7 @@ public Book(String name, String isbn, String picture, String description, String
     }
 
     @Id
-    @GeneratedValue
+    @GeneratedValue(strategy=GenerationType.IDENTITY)
```

Comment:
out of curiosity, what difference does this make to the way the values are generated?

Reply:
Each table will have its own sequence generated for new entities, rather than a global one. This means if you create a customer, then book, then another customer, customer #2 will have id 2 rather than AUTO which will create customer #2 with id 3